src/bundle.rs

```diff
@@ -0,0 +1,89 @@
+use crate::{
+    circuit::Proof,
+    note::{EncryptedNote, NoteCommitment, Nullifier},
+    primitives::tweedlesig::{self, Binding, SpendAuth},
```

Comment:
These are binding and spend auth signatures associated with Pollard, not with Tweedle. IMHO we should try to share as much code as possible between Pollard and Sapling. So, we could define `PollardBinding`, `PollardSpendAuth`, `SaplingBinding`, etc. structs that capture the differences in parameterization, and then we'd be able to use types like `reddsa::PublicKey<PollardBinding>`, etc.

Reply:
We can't reuse RedDSA code unless we use an embedded Edwards curve. Are we doing that, rather than using one of the cycle curves?

Reply:
RedDSA itself doesn't depend on any properties of the curve, other than having a fairly small cofactor (which can be 1). The same code could be generic over any represented group, e.g. any `CofactorGroup` from the `group` crate.